ssl/quic/quic_channel.c

```diff
@@ -1283,8 +1283,10 @@ static int ch_on_transport_params(const unsigned char *params,
     QUIC_CONN_ID cid;
     const char *reason = "bad transport parameter";
 
-    if (ch->got_remote_transport_params)
+    if (ch->got_remote_transport_params) {
+        reason = "multiple transport parameter extensions";
```

Comment:
Can this ever actually happen? I would have expected this to be blocked at the TLS level...but still ok to have this check.

Reply:
You'd know better than me &mdash; I assumed the same TLS extension can appear multiple times in a handshake.